### ISSUE TITLE
How to add/create a new user

### ISSUE THREAD

**Issue by aramaki87** [navidrome/navidrome#1393]
How do I add new/additional users to login at the front end?

I just read: “Multi-user, each user has their own play counts, playlists, favourites, etc...” - but how?

**Comment by lostpolaris** [comment]
presumably, you've already created an admin account. in that case:

in the top right of the page you should see the user icon, once you click on it, you should see the 'users' option
![image](https://user-images.githubusercontent.com/6691735/136085821-318079e9-dcc8-4c22-ad9c-e1ca121ebf46.png)

click on that and you will be led to a page where you can create additional users.

**Comment by aramaki87** [comment]
![5C8DECE2-03DC-4786-B18D-6AF0089D4B7D](https://user-images.githubusercontent.com/17096379/136087690-57ffefb2-a707-407f-8239-e2038b6c79f8.jpeg)

Looks not the same here. If I am not an admin user what is my admin user? 😵‍💫

Using latest version 0.45.1

**Comment by deluan** [comment]
The admin user is the one you created when you first setup Navidrome. You can only add new users if you are logged in with as admin.

If you don't remember the user you created, the only way is to start from scratch again, by stopping Navidrome and deleting the DB files `navidrome.db.*` found in the DataFolder

**Comment by aramaki87** [comment]
I recreated my LXC container and reinstalled everything. Working as intended. Sry for all the trouble and thank you for all your help.

**Comment by deluan** [comment]
Hummm... interesting... seems like a weird bug... I'll reopen this and investigate. Thanks for the report!

**Comment by deluan** [comment]
Sorry, but I was not able to reproduce this.... Do you see any warnings/errors in the logs?

**Comment by aramaki87** [comment]
I tried renaming my admin user. This time everything went alright. I even changed from lower to upper case. Works as intended.

The only difference is the newer version. It was 0.45.1 now I have 0.46.0

The logs show (although it is working) an error: “Authenticated username not found in DB”. I think this is intended and correct the old user is no more and cannot be found in the db and I am redirected to the login page. Login with new user is working.

tl;dr - close this issue. It seems to be fixed by never version. Thank you for all the help. 🙏
